oaknational/Oak-Web-Application
# feat: new lesson listing query - LESQ-793

## Description

Music year: 2019

- updates the lesson listing page to use the new browse mv
- updates tests

## Issue(s)


## How to test

1. Go to https://deploy-preview-2426--oak-web-application.netlify.thenational.academy
Lesson listing pages should look the same (sorry Jack)


## Checklist
